docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,249 @@
+.. _how-to:share:
+
+*****************
+How to share data
+*****************
+
+AiiDA offers two avenues for sharing data with others: archive files and the REST API.
+
+
+.. _how-to:share:archives:
+
+Sharing AiiDA archives
+======================
+
+You have performed your calculations with AiiDA and you would like to share your AiiDA provenance graph, for example to make your scientific study reproducible.
+
+Since AiiDA keeps track of the provenance of every computed result, this step is easy:
+tell AiiDA the **final results** you would like to be reproducible, and AiiDA will automatically include their entire provenance using the :ref:`topics:provenance:consistency:traversal-rules`.
+
+Exporting individual nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Let's say the key results of your study are contained in three AiiDA nodes with PKs ``12``, ``123``, ``1234``.
+Exporting those results together with their provenance is as easy as:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --nodes 12 123 1234
+
+As usual, you can use any identifier (label, PK or UUID) to specify the nodes to be exported.
+
+The resulting archive file ``my-calculations.aiida`` contains all information pertaining to the exported nodes.
+The default traversal rules make sure to include the complete provenance of any node specified and should be sufficient for most cases.
+See ``verdi export create -h`` for ways to modify the traversal rules.
+
+.. tip::
+
+    If you want to make sure that the archive includes everything you intended, you can create a new profile and import it:
+
+    .. code-block:: console
+
+            verdi quicksetup --profile test-export  # create new profile
+            verdi --profile test-export import my-calculations.aiida
+
+            # now use e.g. the QueryBuilder to query the database
+
+Please remember to use **UUIDs** when pointing your colleagues to data *inside* an AiiDA export file, since UUIDs are guaranteed to be universally unique (while PKs aren't).
```

Comment:
Again, please perform a search-and-replace for these cases.
```suggestion
Please remember to use **UUIDs** when pointing your colleagues to data *inside* an AiiDA archive file, since UUIDs are guaranteed to be universally unique (while PKs aren't).
```

Reply:
Did you perform a search-and-replace for these cases? Or are you sure all cases have been addressed?

Reply:
Ok, I'll do a search & replace on the AiiDA codebase

Reply:
I was thinking more within this document (and the documentation in general), but uhm, sure 😅 But maybe this can also wait for a separate PR?

Reply:
I think we can do this once we add the new implementation. We will anyway have to rewrite parts of the documentation. Wouldn't hold up this PR for that

Reply:
Indeed - but it seems @ltalirz is too fast for us both - already went and did it 😮